README.md

```diff
@@ -39,6 +39,9 @@ Run `PORT=8000 yarn start:static` to change the webserver port.
 
 Run `NODE_ENV=development yarn start:static` to build the books (defined in [./.env.development](./.env.development)) and serve them.
 
+
+To prerender the test fixture book and serve it, run `NODE_ENV=test npm run-script prerender && NODE_ENV=test PORT=3000 node ./script/entry server/cli`
```

Comment:
can you not just `NODE_ENV=test yarn serve:static` ?

Reply:
oops, yep! fixed